pylint/checkers/stdlib.py

```diff
@@ -105,6 +107,12 @@ class StdlibChecker(BaseChecker):
                   'The warning is emitted when a threading.Thread class '
                   'is instantiated without the target function being passed. '
                   'By default, the first parameter is the group param, not the target param. '),
+        'W1507': ('Using copy.copy(os.environ). Use os.environ.copy() '  # TODO: number
+                  'instead. ',
+                  'shallow-copy-environ',
```

Comment:
Is this name ok? Shouldn't be rather `shallow-copy-of-environ`?

Reply:
It's good as is :)